Week-1/exercise2.html

```diff
@@ -0,0 +1,205 @@
+            .grid-item a{
+                border: 1px solid;
+                padding: 2px;
+                text-decoration: none;
+            }
+            .footer {
+                position: fixed;
+                left: 0;
+                bottom: 0;
+                width: 100%;
+                height: 10%;
+                background-color: #43689b;
+                color: white;
+                z-index: 2;
+                display: grid;
+                grid-template-columns: 10% 30% 60%;
+            }
+            .grid-item-footer{
+                padding: 10px;
+            }
+            .grid-item-footer img{
+                width: 80px;
+                height: 80px;
+            }
+            .board-actions-footer{
+                padding-top: 3%;
+                float: right;
+                width: 100%;
+            }
+            .grid-item-footer-middle{
+                font-size: 12px;
+                padding-top: 6%;
+                line-height: 0.5;
+                margin-left: -36px;
+            }
+            .board-actions-footer a{
+                text-decoration: none;
+                padding-left: 15%;
+                color: white;
+                font-size: x-large;
+            }
+
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <div class="sidebar">
+                <a href="#"><i class="las la-bars"></i></a>
+                <a href="#"><i class="las la-home"></i></a>
+                <a href="#"><i class="las la-search"></i></a>
+                <a href="#"><i class="las la-volume-up"></i></a>
+                <a href="#"><i class="las la-user"></i></a>
+                <a href="#"><i class="las la-cog"></i></a>
+                <a href="#"><i class="lab la-spotify"></i></a>
+                <a href="#"><i class="lab la-soundcloud"></i></a>
+            </div>
+            <div class="right-content">
+                <div class="info-board">
+                    <div class="info-board-left">
+                        <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="Cattynoard album" />
+                    </div>
+                    <div class="info-board-right">
+                        <div class="board-titles">
+                            <h2>CattyBoard Top 100 Single Charts (11.06.36)</h2>
+                            <p>Unknown Aritist</p>
+                            <p>2016.Charts. 100 songs</p>
+                        </div>
+                        <div class="board-actions">
+                            <a href="#"><i class="las la-play"></i>Play all</a>
+                            <a href="#"><i class="las la-plus"></i>Add to</a>
+                            <a href="#"><i class="las la-ellipsis-h"></i>More</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="main-content-container">
+                    <div class="grid-item even">1.One Dance</div>
```

Comment:
Instead of you deciding what is even and what is odd, Try to use css psedu-selectors